### ISSUE TITLE
Show page number

### ISSUE THREAD

**Issue by naytai** [RocketRobz/TWiLightMenu#517]
This is more a request/enhancement.
It would be nice to have a page indicator like 1/4, 2/4....

**Comment by NightYoshi370** [comment]
Where would the page indicator go though?

**Comment by naytai** [comment]
Maybe this
![4435849569_7ee2e8def5_b](https://user-images.githubusercontent.com/42573484/55722421-a48ea600-59dc-11e9-9201-a358fab74d2f.jpg)

**Comment by Darknior** [comment]
Yes it can ba a good idea ... when we have many roms we don't know how many page there are ... and maybe let us click multiple time on the L o R button to move more than one page at same time.
If we have  10 12 pages let us move faster :)

**Comment by Epicpkmn11** [comment]
Just gonna add another suggestion of the next / previous labels could say the number, like (<kbd>L</kbd> Page 1) (Page 3 <kbd>R</kbd>). This would probably require redoing those graphics as I think the next/previous is text in the image right now, but its the only way I can think of to add numbers without looking out of place
